go.mod

```diff
@@ -6,6 +6,8 @@ replace github.com/hashicorp/consul/api => ./api
 
 replace github.com/hashicorp/consul/sdk => ./sdk
 
+replace launchpad.net/gocheck => github.com/go-check/check v0.0.0-20190902080502-41f04d3bba15
```

Comment:
@likexian is this necessary? I'm wary of adding replace directives to the whole of Consul without knowing exactly why they are needed.

Reply:
on launchpad.net website, it said that gocheck has been moved to github, please check it via: https://launchpad.net/gocheck

Reply:
however I have revert it to avoid the worried.

Reply:
I see. I think it's better to leave it as it was though otherwise we could be pulling in a newer version that the dependency intended. If there is concern about the launchpad version going away then the dependency that imported it should probably have a PR to switch their import rather than us second guessing which one they want here.

Thanks for reverting!